modin/config/pubsub.py

```diff
@@ -228,17 +228,18 @@ def _get_default(cls) -> Any:
         return cls.default
 
     @classmethod
-    def get_value_source(cls) -> Optional[ValueSource]:
+    def get_value_source(cls) -> ValueSource:
         """
         Get value source of the config.
 
         Returns
         -------
-        Optional[ValueSource]
+        ValueSource
         """
         if cls._value_source is None:
             # dummy call to .get() to initialize the value
             cls.get()
+        assert cls._value_source is not None, "_value_source should not be set to None"
```

Comment:
The comment is saying the same thing as the assertion. If you understand the context enough to give a reason why `_value_source` is `None`, I think you should add that.

Reply:
```suggestion
        assert cls._value_source is not None, "_value_source must be initialized by now"
```